tests/nightly/nightly-autorun.sh

```diff
@@ -27,6 +27,10 @@ git fetch https://github.com/vmware/vic master
 git pull
 
 # Kick off the nightly
-now=$(date +"%m_%d_%Y")
-mkdir -p /home/vicadmin/nightly-log/
-sudo -E ./tests/nightly/nightly-kickoff.sh > /home/vicadmin/nightly-log/nightly_$now.txt 2>&1
+echo "Removing VIC directory if present"
+echo "Cleanup logs from previous run"
+
+rm -rf *.zip *.log
+rm -rf bin 60 65
+
+sudo -E ./tests/nightly/nightly-kickoff.sh > ./nightly_console.log 2>&1
```

Comment:
because there is only one console log per run per day, and we clean it up anyways so there is no need to tag it with ${now} and that makes it harder to include in the bundle that way